config/esp32/components/chip/Kconfig

```diff
@@ -130,6 +130,13 @@ menu "CHIP Core"
             help
                 Some device types don't require the read client. Enabling this option may save some flash/ram.
 
+        config ENABLE_PERSIST_SUBSCRIPTIONS
+            bool "Enable persist subscriptions"
+            default n
```

Comment:
The old default was "enabled"...

Reply:
Yes, in our test, if persistent subscriptions are disabled, the HomePod mini could re-establish the subscriptions quickly. So I thinks we could disable this by default.

Reply:
It seems like a PR to change defaults should be separate from this PR, and have a clear description saying what it actually does, etc.